include/pmacc/particles/algorithm/ForEach.hpp

```diff
@@ -0,0 +1,196 @@
+/* Copyright 2017-2019 Axel Huebl, Rene Widera
```

Comment:
@ax3l Are you ok with the license change from GPL (PIConGPU) to LGPL (PMacc). You contributed to parts of the documentation of this class within PIConGPU.

Reply:
Absolutely :+1:

Reply:
Ok for this and all following PRs that move code I authored from PIConGPU to PMacc.